rero_ils/modules/notifications/dispatcher.py

```diff
@@ -84,12 +84,13 @@ def get_dispatcher_function(channel):
             dispatcher_function = get_dispatcher_function(comm_channel)
             counter = len(aggr_notifications)
             if verbose:
-                current_app.logger.info(
-                    f'Dispatch notifications: {notification.type} '
-                    f'library: {notification.library.pid} '
-                    f'patron: {notification.patron.pid} '
-                    f'documents: {counter}'
-                )
+                msg = f'Dispatch notifications: {notification.type} '
+                if hasattr(notification, 'library'):
+                    msg += f'library: {notification.library.pid} '
```

Comment:
I think the preferred way to concat strings is not `msg +=` but `msg = f'{msg} `.